packages/fiori/src/ShellBar.ts

```diff
@@ -1167,6 +1171,7 @@ class ShellBar extends UI5Element {
 				id: `${this.id}-item-${5}`,
 				press: this._handleOverflowPress.bind(this),
 				show: true,
+				tooltip: "Overflow",
```

Comment:
I think we need to translate these hardcoded

Reply:
Sure, it is not OK that way